data/src/main/java/com/example/data/Api.kt

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
You should remove this file, after creating the PR.

Please read this article.
https://www.kenneth-truyers.net/2018/11/01/best-practices-good-pr/#reasonsforrejection